topics/ui/controls/balloon.md

```diff
@@ -12,131 +12,174 @@
+
+![A notification about automatic project configuration: the workspace is restored after a brached is checked out](notification_balloon_timeline_automatic_configuration.png){width=706}
+
+### Suggest an action to configure project or IDE
+
+Use a suggestion notification to promote an action for project or IDE configuration, when not taking this action might lead to less optimal functioning or errors.
+
+![Two examples of a suggestion notification for project or IDE configuration: Windows Defender affecting the IDE with a suggestion to explude a project from it, and a suggestion for switching the UI to an available language](notification_balloon_suggestion_configuration.png){width=706}
+
+### Request user input
+
+Use a suggestion notification to ask the user for an additional input or action that is not connected to their own workflows.
+
+![Two suggestion notifications for requiring user input: a request to fill a survey and a notice about a discount for PhpStorm](notification_balloon_suggestion_request_input.png){width=706}
 
-In other cases, choose another [notification type](notification_types.md).
 
 ## How to use
 
+### Sticky or timed
+
+**Sticky** notifications stay on screen until the user clicks any of its actions or closes it. By default, use this behavior for **suggestion** notifications.
+
+**Timed** notifications stay on screen for 10 seconds and then hide. By default, use this behavior for **timeline** notifications.
+
+Change the default behavior if it makes sense in a particular use case.
+
+
 ### Message severity
 
 #### Error
 
 Use to inform of a critical event or state that might disrupt the user's experience.
 
-![](02_error.png){width=389}
+![An error notification about a plugin being suspended because it requires another plugin that is missing](notification_balloon_severity_error.png){width=706}
 
 #### Warning
 
-Use in case of an event or state that might slow the user's work down or require an action to fix the IDE settings.
+Use in case an event or state might slow the user's work down or require an action to fix the project or IDE settings.
 
-![](03_warning.png){width=389}
+![A warning notification about IDE shortcuts conflicting with macOS shortcuts](notification_balloon_severity_warning.png){width=706}
 
 #### Information
 
-Use to suggest optimizations for the IDE settings and notify about the results of a process that is not tied to any tool window.
-
-![](04_information.png){width=389}
-
-#### Information about a certain plugin
-
-Use to show a plugin-related suggestion.
+Use in all other cases.
 
-Add the plugin’s icon to indicate that the notification relates to a particular plugin. Use a 16×16 SVG icon.
+![An information notification requesting to fill a survey about a plugin](notification_balloon_severity_info.png){width=706}
 
-![](05_information_plugin.png){width=389}
+When possible, use a plugin or functionality icon instead of the info icon. This helps identifing the source of the notification quicker.
```

Comment:
Can we add a title here as well?
Also, I noticed there is a balloon with an emoji for the PhpStorm discount, but no indication on when to use emoji or other icons? Not sure if it's necessary here but an information about them would definitely eliminate some corner cases